e2e_playwright/st_components_v1.py

```diff
@@ -14,6 +14,12 @@
 
 import streamlit as st
 import streamlit.components.v1 as components
+from streamlit.runtime.scriptrunner import get_script_run_ctx
+
+ctx = get_script_run_ctx()
+if ctx is None:
+    # TODO(lukasmasuch) This script is not compatible with running it in "bare" mode (e.g. `python script.py`)
+    exit(0)
```

Comment:
Should we (also) print a log here?

Reply:
I hope that we can remove this in a quick follow-up once we fix the components issue with bare mode.